utils/installer/install.sh

```diff
@@ -296,6 +302,8 @@ function __validate_node_installation() {
 
   if [ "$pkg_manager" == "npm" ]; then
     manager_home="$(npm config get prefix 2>/dev/null)"
+  elif [ "$pkg_manager" == "bun" ]; then
+    manager_home="$(echo "$BUN_INSTALL" 2>/dev/null)"
```

Comment:
any reason this can't be just this?
```suggestion
    manager_home="$BUN_INSTALL"
```

Reply:
good point